lib/bitters/generator.rb

```diff
@@ -76,5 +85,27 @@ def stylesheets_directory
     def top_level_directory
       File.dirname(File.dirname(File.dirname(__FILE__)))
     end
+
+    def command_exists?(command)
+      `which #{command}` != ""
+    end
+
+    def change_file_ext(file, ext)
+      no_ext = File.join(File.dirname(file), File.basename(file, ".*"))
+      "#{no_ext}.#{ext}"
+    end
+
+    def convert_syntax_to_sass
+      if command_exists?("sass-convert")
```

Comment:
I would swear this was working for me before, but now the `command_exists?` method isn't working. I switched to using `which` like so and it worked for me:
```
if system("which sass-convert > /dev/null 2>&1")
```